packages/mapviewer/src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -181,11 +181,10 @@ function getIframeHosts(value) {
 :global(.t_list) {
     width: 100%;
 }
-td {
+:global(td) {
```

Comment:
I don't know if we should maybe try to scope that a bit more, here we are applying that to ALL `<td>` on the app.
Could we use something like `:global(.htmlpopup-container td)` to reduce a bit the scope of this rule?